notes/klustered-teams/episode-3/skyscanner/README.md

```diff
@@ -0,0 +1,33 @@
+# Kluster 000
+
+## Discovered Symptoms
+
+
+## Contributing Factors
+
+
+## Notes from Kluster Breaker
+
+### APIServer Break
+
+We modified the core `v1.` [apiservice](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-aggregation-layer/#register-apiservice-objects) object to point to the `kubernetes` service in the `default` namespace, resulting in the APIServer being unable to ever successfully break out of a loop and respond as all API requests got trapped in a loop. The reason restarting the APIServer was enough to fix this was that the built-in aggregator server which manages aggregation of APIs within kubernetes rebuilds this list on startup from the API resources defined within the Kube-APIServer.
```

Comment:
More than breaking out of the loop, wasn't because the API server can't serve those requests over the service? There is probably a difference between `Local` and the service we added.

Then I have a small suggestion
```quote
Restarting the APIServer fixed the issue because the built-in aggregator server - which manages aggregation of APIs within kubernetes - rebuilds this list on startup from the API resources defined within the Kube-APIServer.
```

Reply:
You're right, yep.

I've just run through this with a kind cluster again with some verbose logging and the endpoints for the kubernetes service in the default namespace resolve to the same IP and port that Kubernetes is advertising, but the APIserver first gets stuck trying to do OpenAPI discovery against the new endpoints and proceeds to have many TLS Errors:

`error trying to reach service: x509: certificate signed by unknown authority` after a bit of this, it forces a resync of all the endpoints, still fails, and is then unable to renew its lease:

`unable to create required kubernetes system namespace kube-node-lease: the server is currently unable to handle the request (post namespaces)`

This is the point where it starts returning the `service unavailable` we saw.

Reply:
I've updated this section now, what do you think?